pages/components/Header.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import styles from './Header.module.css';
+
+const Header = () => {
+  return (
+    <header className={styles.header}>
+        <nav>
+            <a href='/' className={styles.nav_item}>Home</a>
```

Comment:
As mentioned in the **Note** section of the issue a navigation bar is a list of links. In this case, the list is missing. Please check out [MDN doc on Nav](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) for further information.

Reply:
List is added.